test/topology_experimental_raft/test_tablets.py

```diff
@@ -1497,3 +1502,29 @@ async def assert_empty_table():
 
     logger.info("Verify data is not resurrected")
     await assert_empty_table()
+
+@pytest.mark.repair
+@pytest.mark.asyncio
+async def test_tablet_manual_repair(manager: ManagerClient):
+    servers, cql, hosts, table_id = await create_table_insert_data_for_repair(manager, fast_stats_refresh=False)
+    token = -1
+
+    start = time.time()
+    await manager.api.tablet_repair(servers[0].ip_addr, "test", "test", token)
+    duration = time.time() - start
+    map1 = await load_tablet_repair_time(cql, hosts[0:1], table_id)
+    logging.info(f'map1={map1} duration={duration}')
+
+    await asyncio.sleep(1)
```

Comment:
Why is this sleep needed?

Reply:
Because the repair time granularity is seconds. This makes sure the second repair is different than the previous one.

Reply:
I will add a comment in the test.

Reply:
Ok